testing/integration-testing-hooks.ts

```diff
@@ -458,7 +461,17 @@ export function skipIfEnvServerVersion(semverCondition: string): void {
   before(async function() {
     let testServerVersion = process.env.MONGOSH_SERVER_TEST_VERSION;
     if (!testServerVersion) {
-      testServerVersion = await getInstalledMongodVersion();
+      try {
+        testServerVersion = await getInstalledMongodVersion();
+      } catch(e) {
+        // no explicitly specified version but also no local mongod installation
+        // -> spawn a server to get the "live" version
```

Comment:
What is the 'live' version here :)?

Starting a mongod server just for the version probably is unnecessary.

My understanding of how the tests are setup here is that mlaunch without an env var would:

1. try to see if a mongod binary is in the path an use it
2. if not download and start the latest

In that case maybe we could do something like this (super-inaccuarte-pseudo-code):

``` js
const ver = await commandExists('mongod')? parseVersion(await exec('mongod --version')) : 9999.9999.9999
```

Reply:
The "live" version is the one we really end up using :D But good point I could default to a super high version number since it is the "latest".

Shutdown is not necessary as it is a "shared" instance and will be re-used and later shut down automatically.

Reply:
Oh nice!

Reply:
> since it is the "latest"
Oh is always latest? When i run it locally it uses the mongod in the path, which i have as 4.0. 

Anyway if this approach works better then we can also merge that.

Reply:
Yeah _if_ it finds one on the path, mlaunch will use that. Otherwise it downloads `latest`.

But that's unfortunately exactly the corner case: there is nothing on the path and nothing in the env vars. So we "dynamically" end up with what gets downloaded for `latest`. So in the case here it will work even with the hard-coded "9999.9999.9999" version since I'm comparing against a "past version". I'll hard code that version and put it in the comment since needing this function is already an edge case by itself 😅 
We only need this since the server parameters changed from one version to the other inside our testing matrix...

Reply:
I see ok!

Reply:
Ping me if you need another review, going to pre-approve it so you can merge if i'm not around